Source/App.js

```diff
@@ -1 +1,417 @@
-var cesiumWidget = new Cesium.Viewer('cesiumContainer');
+    //////////////////////////////////////////////////////////////////////////
+    // Viewer configuration
+    //////////////////////////////////////////////////////////////////////////
+
+    var viewer = new Cesium.Viewer('cesiumContainer', {
+        navigationHelpButton: false,
```

Comment:
I wouldn't disable this.  No one knows how to use our mouse controls the first time they bring up Cesium